activerecord/lib/active_record/connection_adapters/abstract/transaction.rb

```diff
@@ -333,6 +346,10 @@ def commit_records
       def full_rollback?; true; end
       def joinable?; @joinable; end
 
+      def fixture_transaction?
```

Comment:
@matthewd @byroot I've opted for a method like this rather than a subclass because I think that's simpler than making it fit under `Transaction` / `SavepointTransaction` / `RealTransaction`.